manim/utils/debug.py

```diff
@@ -16,12 +20,62 @@ def print_family(mobject, n_tabs=0):
         print_family(submob, n_tabs + 1)
 
 
-def index_labels(mobject, label_height=0.15):
+def index_labels(
+    mobject: "Mobject",
```

Comment:
If it is imported anyways there is no need to use a string in the type hints (I think)

```suggestion
    mobject: Mobject,
```